org.eclipse.lsp4xml.extensions.maven/src/main/java/org/eclipse/lsp4xml/extensions/maven/searcher/RemoteRepositoryIndexSearcher.java

```diff
@@ -125,7 +129,10 @@ public void transferCompleted(TransferEvent transferEvent) {
 		}
 	}
 	
-	private Set<String> internalGetArtifactVersions(Artifact artifactToSearch, String packaging, IndexingContext... requestSpecificContexts) {
+	private Set<ArtifactVersion> internalGetArtifactVersions(Dependency artifactToSearch, String packaging, IndexingContext... requestSpecificContexts) {
+		if (artifactToSearch.getArtifactId() == null || artifactToSearch.getArtifactId().trim().isEmpty()) {
```

Comment:
It might be worth also checking if the groupId is null or empty, as we use the groupId on line [136](https://github.com/AObuchow/lsp4xml-extensions-maven/pull/67/files#diff-644fbb2cadadc069cea44da435e4633bR136)

Reply:
Maybe in the same way it's done [here](https://github.com/AObuchow/lsp4xml-extensions-maven/pull/67/files#diff-644fbb2cadadc069cea44da435e4633bR166)